### PR TITLE
OSDOCS-11920: Added OSD GCP Architecture models page.

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-11920

Link to docs preview:
[Architecture models](https://83930--ocpdocs-pr.netlify.app/openshift-dedicated/latest/architecture/osd-architecture-models-gcp.html)

QE review:
No QE approval needed for this change.

Peer review:
- [x] Peer reviewer has approved this change.

SME review:
- [x] SME has approved this change.

Additional information:

### CODE REVIEW

File: _topic_maps/_topic_map_osd.yml [hunk 1]
```diff
@@ -74,6 +74,8 @@ Topics:
   File: index
 - Name: Product architecture
   File: architecture
+- Name: Architecture models
```
Comment (bng0y) [inline _topic_maps/_topic_map_osd.yml:77]: Could we add GCP to the Name as well

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Comment by AedinC** [comment]
> Just a couple of comments: I do wonder if we could reduce the size of the titles in the TOC? Maybe fine, and will have to stick with what is currently shown.

Yeah agree it could look better! Think the inclusion of the PSC module has messed things up a bit. I will have a play around today to see if I can improve things-thanks!

**Comment by svmrh** [comment]
On the [architecture models](https://83930--ocpdocs-pr.netlify.app/openshift-dedicated/latest/architecture/osd-architecture-models-gcp.html) page, can we update the following NOTE with what we have in the OCM UI. This is to highlight the customer benefits when using PSC - "_**Red Hat recommends using Private Service Connect when deploying a Private OpenShift Dedicated cluster on Google Cloud. Private Service Connect ensures there is a secured, private connectivity between Red Hat infrastructure, Site Reliability Engineering (SRE) and private OpenShift clusters.**_"

> Red Hat recommends the use of Private Service Connect-based architecture when deploying private OpenShift Dedicated clusters on GCP.

image

**Comment by svmrh** [comment]
In [Private OpenShift Dedicated on Google Cloud Platform (GCP) **without** Private Service Connect (PSC)](https://83930--ocpdocs-pr.netlify.app/openshift-dedicated/latest/architecture/osd-architecture-models-gcp.html#osd-private-architecture-model_osd-architecture-models-gcp) architecture model sub-topic, can we add the same recommendation as above? This is to nudge customers to use PSC when deploying private clusters. 

> "Red Hat recommends using Private Service Connect when deploying a Private OpenShift Dedicated cluster on Google Cloud. Private Service Connect ensures there is a secured, private connectivity between Red Hat infrastructure, Site Reliability Engineering (SRE) and private OpenShift clusters."

**Comment by svmrh** [comment]
Other than the above two comments, [Architecture models](https://83930--ocpdocs-pr.netlify.app/openshift-dedicated/latest/architecture/osd-architecture-models-gcp) page looks good.

**Pull request merged**